### ISSUE TITLE
impossible to insert definitions in the YAML

### ISSUE THREAD

**Issue by kickTiger** [onlyafly/number-headings-obsidian#54]
Because in version 1.4.5, new attribute functionality was added.

I often found it impossible to insert definitions in the YAML area.
Even if I forcibly wrote definition code in the YAML area, it would not automatically sort
YAML

```yaml
number headings: auto, first-level 2, max 1, start-at 1, 1.1
```

This impact is too serious. This plugin is the one I use the most, and I hope you can check and reproduce this issue and get it resolved. Thank you very much.

**Comment by kickTiger** [comment]
I've rechecked and managed to replicate the issue you described. Here are the conditions:

- I installed only **number-headings-obsidian** plugin in my Sandbox vault.
- In Live Preview, I invoked the command via the command palette and selected "number headings: number all headings in document."
- I noticed that I couldn't insert content into the new properties area, which is the YAML area.
- **Cannot be inserted** if the yaml region already has properties .

It is possible to insert if the yaml region is blank, or if there is no number-headings-obsidian properties 


I've recorded a GIF animation to provide you with a visual reference.

![number-headings-obsidian](https://github.com/onlyafly/number-headings-obsidian/assets/3623068/00b68d81-e5c8-4757-9961-122b4ddded9b)
